### DISCUSSION TITLE
Feature Request: Make "Refresh Dashboard" Event over Global Event List available in Panel Plugin

### DISCUSSION THREAD

**Discussion by mudumat** [grafana/grafana#38292] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
At the moment as far I know its just possible to emit a Dashboard Refresh by an ugly hack to do a virtual click on the dashboard refresh button. Additional it would be nice to have a possibility to register custom events and listen to them and ask for a list of events that got registered globally. 

**Why is this needed**:
After changing a activity status in a custom panel that has to raise an update for all graph panels. It would  make communication between several panels possible.

As an example I attached the status change and forced refresh by the ugly refresh dashboard hack.

https://user-images.githubusercontent.com/2966967/120812616-6f932500-c54d-11eb-83c6-d29fb9f0602e.mp4

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
